requirements.txt

```
@@ -25,7 +25,7 @@ Pillow==9.0.1  # min version = 7.0.0 (Focal with security backports)
 polib==1.1.0
 psutil==5.6.7 # min version = 5.5.1 (Focal with security backports)
 psycopg2==2.7.7; sys_platform != 'win32' and python_version < '3.8'
-psycopg2==2.8.6; sys_platform == 'win32' or python_version >= '3.8'
+psycopg2==2.9.5; sys_platform == 'win32' or python_version >= '3.8'
```

The libpq dll will be missing when psycopg2 is not installed via a wheel package 
https://www.psycopg.org/docs/install.html#runtime-requirements

The reason why this is not working is because there is no windows wheel package for python > 3.9.
https://pypi.org/project/psycopg2/2.9.5/#files

It is working on linux because we are installing the libpq manually and all requirements are met.
https://www.odoo.com/documentation/16.0/administration/install/install.html#python

So to avoid changing the version of psycopg2 for linux, I would suggest to only adapt the requirement for windows.

```suggestion
psycopg2==2.8.5; sys_platform != 'win32' and python_version >= '3.8'
psycopg2==2.9.5; sys_platform == 'win32' # wheel package 2.8.5 not available for python > 3.9
```

A question remains, should we also manage the case of 
psycopg2==2.8.5; sys_platform == 'win32' and python_version < '3.9'. But this is maybe not a bid deal to change the version for windows only.